scripts/helpers/todos.js

```diff
@@ -76,7 +76,7 @@ function writeTodos(file, todos) {
     content += `# ${item.pkg}\n`;
 
     for (const [type, todos] of item.contentByTypeGrouped) {
-      content += `## ${type}\n`;
+      content += `### ${type}\n`;
```

Comment:
Headings should be logically nested and no levels skipped (https://www.w3.org/WAI/WCAG21/Techniques/general/G141.html).